src/documentation/transxchange_read.md

```diff
@@ -107,16 +130,68 @@ Similarly, the element *BankHolidaysOperation* may be also be present, specifyin
+trip_id | *VehicleJourney/ServiceRef*, *VehicleJourney/LineRef*, *VehicleJourney/VehicleJourneyCode* | This field is prefixed and formed by the concatenation of the three elements separated by a `:` in order to guarantee uniqueness.
+trip_headsign | *JourneyPatternTimingLink/DestinationDisplay* or *VehicleJourneyTimingLink/DestinationDisplay* | In case both elements are specified, the *VehicleJourney* overrides the *JourneyPattern*. In case none of the elements is specified, *Services/Service/StandardService/JourneyPattern/DestinationDisplay* should be used. Otherwise, the field is left empty.
+company_id | *VehicleJourney/OperatorRef* | This field is prefixed. Link to the file [companies.txt](#companiestxt). The referenced *Operators/Operator/OperatorCode* is used. If no *OperatorRef* is specified for the trip, the associated *Services/Service/RegisteredOperatorRef* is used to retrieve the company for the trip.
+physical_mode_id | *Services/Service/Mode* | This field is not prefixed. Link to the file physical_modes.txt of the NTFS. See above for the mapping of transport modes.
+trip_properties.wheelchair_accessible | *VehicleJourney/Operational/VehicleType/WheelchairAccessible* | The value is `1` when the trip is accesible, `2` when the trip is not accessible and `0` when the field is not specified.
 
 ### stop_times.txt
+The passing times at each stoppoint of a trip are specified as an ordered list of 
+links between the stoppoints (*JourneyPatternTimingLink*s) in the *JourneyPatternSection* attached to the associated *JourneyPattern*.
+
+In some (rare) cases, a *VehicleJourney* might specify explicitly some timing links 
+that are different from the underlying *JourneyPattern*. In this case, a 
+*VehicleJourneyTimingLink* overrides any common property it shares with a *JourneyPatternTimingLink*.
+
+#### Computing passing times at each stoppoint
+The arrival/departure time for the first stoppoint of a trip is explicitly specified 
+in *VehicleJourney/DepartureTime*. For each subsequent stoppoint, the passing time is 
+calculated from the cumulative sum of the current *JourneyPatternTimingLink* values 
+for all preceding stops in the journey link sequence as follows:
+- arrival_time at stoppoint<sub>n</sub> = departure_time from stop<sub>n-1</sub> + (*RunTime* for inbound link from stop<sub>n-1</sub>)
+- departure_time at stop<sub>n</sub> = arrival_time at stop<sub>n</sub> + *WaitTime* for destination end of inbound link from stop<sub>n-1</sub>) + *WaitTime* for origin of outbound link to stop<sub>n+1</sub>
+
+If *WaitTime* is not specified, the default waiting time at a stoppoint is considered `0`.
+
+Note that *RunTime* and *WaitTime* are given as [durations](https://en.wikipedia.org/wiki/ISO_8601#Durations).
+
+Except from the first stoppoint of the trip, the arrival end (*JourneyPatternTimingLink/To*) 
```

Comment:
We reversed with the use of JourneyPatternTimingLink/From, so the except is on last stop.
No problem, the result is the same.